### ISSUE TITLE
Usage Data Sharing cron job causes Fatal Error on sites

### ISSUE THREAD

**Issue by askwpgirl** [elementor/elementor#29153]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

When Usage Data Sharing is enabled, a cron job hooks=elementor%2Ftracker%2Fsend_event  is triggered once a week. This cron job throws a fatal error:

`PHP Fatal error:  Uncaught Error: Call to undefined function Elementor\Core\get_plugins() in /www/wp-content/plugins/elementor/core/wp-api.php:26`

This code pertains to the get_plugins function in the wp-api.php file:

`$this->plugins = new Collection( get_plugins() );`

Perhaps this needs to be checked per the documentation:

https://developer.wordpress.org/reference/functions/get_plugins/

As per the documenation:

> If you have `PHP Fatal error: Call to undefined function get_plugins()` then you must include the file ‘wp-admin/includes/plugin.php‘ like in example.

See the example in the documentation.

This error does not occur when Usage Data Sharing is disabled.

### Steps to reproduce

1. Enable Usage Data Sharing under Elementor > Settings.
2. Wait for cron job elementor%2Ftracker%2Fsend_event to run once a week.
3. Check PHP error log.

### Expected behavior

This cron job should not throw a fatal error per the get_plugins WordPress developer documentation.

### Isolating the problem

- [X] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

```txt
== Server Environment ==
	Operating System: Linux
	Software: nginx/1.25.3
	MySQL version: mariadb.org binary distribution v10.3.17
	PHP Version: 7.4.33
	PHP Memory Limit: 768M
	PHP Max Input Vars: 5000
	PHP Max Post Size: 500M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.6.2
	Site URL: 
	Home URL: 
	WP Multisite: No
	Max Upload Size: 500 MB
	Memory limit: 768M
	Max Memory limit: 768M
	Permalink Structure: /%postname%/
	Language: en_US
	Timezone: Pacific/Honolulu
	Admin Email: 
	Debug Mode: Inactive

== Theme ==
	Name: GeneratePress
	Version: 3.4.0
	Author: Tom Usborne
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36

== Active Plugins ==
	Code Snippets
		Version: 3.6.5.1
		Author: Code Snippets Pro

	Elementor
		Version: 3.25.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.25.1
		Author: Elementor.com

	Image Optimizer - Compress, Resize and Optimize Images
		Version: 1.5.4
		Author: Elementor.com

	Video User Manuals
		Version: 2.5.14
		Author: Video User Manuals Pty Ltd


== Must-Use Plugins ==
	SiteDistrict Single Sign-on
		Version: 0.1.0
		Author: SiteDistrict Team


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Improved CSS Loading: Inactive by default
	Inline Font Icons: Inactive by default
	Additional Custom Breakpoints: Active by default
	Container: Inactive by default
	Upgrade Swiper Library: Active by default
	Nested Elements Performance: Active by default
	Optimized Control Loading: Active by default
	Optimized Markup: Inactive by default
	Conditionally load Swiper CSS files: Inactive by default
	Plugin Onboarding: Active by default
	CSS Smooth Scroll: Active by default
	Default to New Theme Builder: Inactive
	Elementor Home Screen: Active by default
	Landing Pages: Inactive
	Nested Elements: Active by default
	Editor Top Bar: Active by default
	Pages Panel: Inactive by default
	Element Caching: Inactive by default
	Link In Bio: Active by default
	Floating Buttons: Active by default
	Elementor Editor Events: Inactive by default
	Atomic Widgets: Inactive by default
	Launchpad Checklist: Inactive by default
	Menu: Inactive by default


== Log ==
	
Log: showing 20 of 402024-09-12 14:16:36 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.24.0',
  'to' => '3.24.1',
)]
2024-09-13 02:22:52 [info] elementor-pro::elementor_pro_updater Started 
2024-09-13 02:22:52 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-09-13 02:22:53 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-09-13 02:22:53 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.24.0',
  'to' => '3.24.1',
)]
2024-09-13 02:22:53 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.24.0',
  'to' => '3.24.1',
)]
2024-09-18 14:16:37 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.24.1',
  'to' => '3.24.2',
)]
2024-10-31 11:06:15 [info] elementor::elementor_updater Started 
2024-10-31 11:06:15 [info] Elementor/Upgrades - _on_each_version Start  
2024-10-31 11:06:15 [info] Elementor/Upgrades - _on_each_version Finished 
2024-10-31 11:06:15 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.24.0',
  'to' => '3.25.3',
)]
2024-10-31 11:06:17 [info] elementor-pro::elementor_pro_updater Started 
2024-10-31 11:06:17 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-10-31 11:06:17 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-10-31 11:06:17 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.24.1',
  'to' => '3.25.0',
)]
2024-10-31 14:16:37 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.25.0',
  'to' => '3.25.1',
)]
2024-10-31 14:45:28 [info] elementor-pro::elementor_pro_updater Started 
2024-10-31 14:45:28 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-10-31 14:45:28 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-10-31 14:45:28 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.25.0',
  'to' => '3.25.1',
)]

JS: showing 1 of 1JS: 2024-08-26 15:50:46 [error X 1][https://______.net/wp-admin/post.php?post=8&amp;action=elementor:3:16829] Cannot read properties of undefined (reading &#039;localized&#039;) 

PHP: showing 2 of 2PHP: 2024-08-29 14:29:43 [error X 9][/www/wp-content/plugins/elementor/core/wp-api.php::26] Uncaught Error: Call to undefined function Elementor\Core\get_plugins() in /www/wp-content/plugins/elementor/core/wp-api.php:26
Stack trace:
#0 /www/wp-content/plugins/elementor/modules/compatibility-tag/base-module.php(96): Elementor\Core\Wp_Api-&gt;get_plugins()
#1 /www/wp-content/plugins/elementor/modules/compatibility-tag/base-module.php(58): Elementor\Modules\CompatibilityTag\Base_Module-&gt;get_plugins_with_header()
#2 /www/wp-content/plugins/elementor/modules/compatibility-tag/module.php(57): Elementor\Modules\CompatibilityTag\Base_Module-&gt;get_plugins_to_check()
#3 /www/wp-content/plugins/elementor/modules/compatibility-tag/base-module.php(148): Elementor\Modules\CompatibilityTag\Module-&gt;get_plugins_to_check()
#4 /www/wp-includes/class-wp-hook.php(324): Elementor\Modules\CompatibilityTag\Base_Module-&gt;Elementor\Modules\CompatibilityTag\{closure}()
#5 /www/wp-includes/class-wp-hook.php(348): WP_Hook-&gt;apply_filters()
#6 /www/wp-includes/plugin.php(517): WP_Hook-&gt;do_action()
#7 /www/wp-content/plugins/elementor/ [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-08-29 18:37:15 [notice X 2][/www/wp-content/plugins/elementor/includes/managers/image.php::113] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
```

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by louiswol94** [comment]
Thanks Angela! @askwpgirl

**Comment by nicholaszein** [comment]
Hi Angela @askwpgirl!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved with the release of Elementor v3.26.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
